src/lua/skiller/subskill_jumpstate.lua

```diff
@@ -228,20 +230,44 @@ function SubSkillJumpState:do_init()
 
    self:skill_reset()
    self.skill_status = skillstati.S_RUNNING
-
+   
    self.args = {}
+
+   -- store expected size of arguments table to compare after run of init()
+   local exp_args = {}
+   local act_args = {}
+
```

Comment:
Please put a comment here that explains what is happening below. Something like "initialize empty self.args table for every subskill". This (existing) code is quite verbose and I was having a little difficulty understanding excactly how your patch affects the behaviour.